dspace/config/spring/api/discovery.xml

```diff
@@ -565,7 +566,7 @@
         <property name="defaultFilterQueries">
             <list>
                 <!--Only find items, communities and collections-->
-                <value>search.resourcetype:2 AND relationship.type:Publication</value>
+                <value>search.resourcetype:2 AND entityType_keyword:Publication</value>
```

Comment:
There are several instances of this rename relationship.type -> entityType_keyword also included in this patch.  Was this intentional?  The change is not mentioned.

Reply:
Good point @mwoodiupui ! It's more clear to me what `relationship.type:Journal` stands for comparing it with `entityType_keyword`. @benbosman can you explain why the renaming? Thank you. Besides this, I don't have any more comments. Thank you @benbosman !

Reply:
Agreed, this change to `entityType_keyword` doesn't make sense to me either...it looks almost accidental?

Reply:
Sorry I missed that in my description.
This is actually just the indexed field for `relationship.type` as defined in https://github.com/DSpace/DSpace/blob/master/dspace/config/spring/api/discovery.xml#L1061
The reason to switch to this value was due to an error we noticed with the filter `relationship.type:Journal` which also contained items with value JournalVolume and JournalIssue.
I'm updating the description accordingly

Reply:
Yes, that's default DSpace behavior.
https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/discovery/SolrServiceImpl.java#L1600  indexes the value.
https://github.com/DSpace/DSpace/blob/master/dspace/solr/search/conf/schema.xml#L289 ensures this is indexed with a keywordFilter, which treats the entire field as a single token, regardless of its content